CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Support for the `LLVM` symbol, which is defined on LLVM-based toolchains from Delphi 12 onward.
 - Support for the `IOSSIMULATOR` symbol, which is defined on the `DCCIOSSIMARM64` toolchain.
+- `FormDfm` analysis rule, which flags VCL forms/frames lacking a `.dfm` resource.
+- `FormFmx` analysis rule, which flags FireMonkey forms/frames lacking a `.fmx` resource.
+- **API:** `ResourceDirective` type, which can be returned by the `CompilerDirectiveParser`.
```

Comment:
This is very specific, but not very descriptive - I don't think anyone reading the changelog
will understand what this means in a concrete sense. Maybe something like

```markdown
- **API: `CompilerDirectiveParser` now returns a new `ResourceDirective` type when parsing resource directives (`{$R}`)
```

Reply:
Ended up going for:
- **API: `CompilerDirectiveParser` now returns a new `ResourceDirective` type when parsing 
  [resource directives](https://docwiki.embarcadero.com/RADStudio/en/Resource_file_(Delphi)).